ONSdigital/eq-questionnaire-validator
# Upgrade to Node V20 and drop support for Yarn in Validator

### What is the context of this PR?
Upgrade Node to version v20.10.0 and drop support for Yarn. Upgraded because Node.js 20 introduces important updates, bug fixes, and performance enhancements, ensuring our application remains up to date and secure.

### How to review
- Ensure `Yarn` has been removed and check the repository for any yarn that should not be there
- Ensure new `npm` commands work as previously

### Checklist
